backend/api/EventHandlers/MqttEventHandler.cs

```diff
@@ -710,31 +710,6 @@ private async void OnIsarStartup(object? sender, MqttReceivedArgs mqttArgs)
                 robot.Name,
                 robot.IsarId
             );
-            try
-            {
-                await MissionScheduling.MoveCurrentMissionRunBackToQueue(robot.Id);
```

Comment:
Do we have an alternative way to handle "re-scheduling" of missions when isar restarts?

Reply:
No, the mission would be considered failed, and ISAR should stop the mission. If we want to reschedule it, then ISAR can send a "mission aborted" MQTT message.